app/controllers/applications/process_controller.rb

```diff
@@ -114,6 +114,11 @@ def summary
     def summary_save
       ResolverService.new(application, current_user).complete
       redirect_to application_confirmation_path(application.id)
+    rescue ActiveRecord::RecordInvalid => ex
+      flash[:alert] = I18n.t('error_messages.summary.validation')
+      Raven.capture_exception(ex, application_id: application.id)
+
+      redirect_to :back
```

Comment:
I see in the specs we [expect](https://github.com/ministryofjustice/fr-staffapp/blob/a9f016e091a9864b8651ad1b2701a95476d7d9ad/spec/controllers/applications/process_controller_spec.rb#L488) the user to get redirected to the root url. Maybe it would be better to do the same here as well because `:back` could be anything.